Liang-README HW1.md

```diff
@@ -0,0 +1,9 @@
+## General Introduction about Myself
+My name is Feng Liang and I am a MAPSS student in sociology. My thesis is very qualitative oriented and almost does not involve any manipulation of numeric data at all. It is very refreshing for me to sit in this class and to think about new ways of interacting with data. Here's link to the [Markdown guide](https://guides.github.com/features/mastering-markdown/#examples) that is **enormously** helpful for my assignment.
```

Comment:
Best practice is to leave a line break between different types of content. For example, leave a blank line between a header and a block of text.

```
## General Introduction about Myself
My name is Feng Liang and I am a MAPSS student in sociology. My thesis is very qualitative oriented and almost does not involve any manipulation of numeric data at all. It is very refreshing for me to sit in this class and to think about new ways of interacting with data. Here's link to the [Markdown guide](https://guides.github.com/features/mastering-markdown/#examples) that is **enormously** helpful for my assignment.
```

It makes it easier to read the original `.md` file, plus also prevents rendering errors later trying to view the document as an HTML or PDF file.